os/kernel/log_dump/log_dump.c

```diff
@@ -239,3 +440,48 @@ size_t log_dump_read(FAR char *buffer, size_t buflen)
 
 	return ret;
 }
+
+int log_dump(int argc, char *argv[])
+{
+	struct mq_attr attr;
+	mqd_t mq_fd;
+
+	attr.mq_maxmsg = 32;
+	attr.mq_msgsize = sizeof(struct log_dump_msg_s);
```

Comment:
Please check if this needs to be changed to ```sizeof(int)``` as the message being passed below is of type int.

Reply:
done